ksql-api/src/test/java/io/confluent/ksql/api/TestEndpoints.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2019 Confluent Inc.
```

Comment:
All the tests seem to also have 2019 in copyright

Reply:
Ack